pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1217,6 +1217,13 @@ self: super: {
     })];
   });
 
+  # Remove unecessary constraint
```

Comment:
Please report this issue upstream and attach the URL of the ticket/GH issue here in a comment.

Reply:
Ok, I'll do.